src/Exception/ActionUnavailableException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Opstalent\CrudBundle\Exception;
+
+use Throwable;
+
+/**
+ * Class UnavailableActionException
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle\Exception
+ */
+class ActionUnavailableException extends \LogicException implements Exception
```

Comment:
i prefer to add all classes in use statements: \LogicException → LogicException but up to you